xds/internal/balancer/clusterresolver/configbuilder_test.go

```diff
@@ -957,17 +605,16 @@ func assertString(f func() (string, error)) string {
 	return s
 }
 
-func testAddrWithAttrs(addrStr string, weight *uint32, priority string, lID *internal.LocalityID) resolver.Address {
+func testAddrWithAttrs(addrStr string, localityWeight uint32, endpointWeight uint32, priority string, lID *internal.LocalityID) resolver.Address {
```

Comment:
Nit: `localityWeight, endpointWeight uint32` instead.

Reply:
Done. Will keep in mind for future.